benchmark.sh

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env bash
```

Comment:
One final call out. This is not currently checked in with the executable bit set as I'm not sure what the projects preference would be for that.